config.go

```diff
@@ -36,6 +36,7 @@ func parseFlags() {
 	flag.String("localstatedir", "", "Directory to store local state information. (default: workdir)")
 	flag.String("rundir", "", "Directory to store runtime data. (default: workdir/run)")
 	flag.String("logdir", "", "Directory to store logs. (default: workdir/log)")
+	flag.String("logfile", "-", "Log file name. (default: -)")
```

Comment:
Sure. Will make glusterd2.log as default. and accept `stderr` or `stdout` as inputs and use os.Stderr or os.Stdout respectively.

    --logfile=stderr
    --logfile=stdout
    --logfile=somelog.log